recipes/visp/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set name = "visp" %}
+{% set version = "3.5.0" %}
+{% set major_minor_version = version.split('.')[0] + version.split('.')[1] + version.split('.')[2] %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lagadic/visp/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 494a648b2570da2a200ba326ed61a14e785eb9ee08ef12d3ad178b2f384d3d30
+  patches:
+    - patches/0001-Fix-invalid-usage-of-OGRE_DIR-in-CMakeLists.txt.patch
+    - patches/0002-Disable-AVX-and-AVX2-for-Simd-embedded-lib.patch  # [win]
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - clang  # [win]
+    - make  # [not win]
+    - ninja  # [win]
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}  # [linux]
+  host:
+    - xorg-libx11  # [unix]
+    - xorg-libxfixes  # [unix]
+    - libxml2  # [unix]
+    - libdc1394 >=2.2.6  # [unix]
+    - librealsense  # [unix]
+    - _openmp_mutex  # [linux]
+    - libopenblas  # [osx]
+    - llvm-openmp  # [win or osx]
+    - openmp  # [win]
+    - pthreads-win32  # [win]
+    - libopencv
+    - eigen
+    - libjpeg-turbo
+    - libpng
+    - ogre >=1.9.0,<=1.10.12
+    - mkl-devel
+
+  run:
+    - pthreads-win32  # [win]
+    - mkl
+
+test:
+  commands:
+    - test -f $PREFIX/include/visp3/visp.h  # [unix]
+    - test -f $PREFIX/lib/cmake/visp/VISPConfig.cmake  # [unix]
+    - test -f $PREFIX/lib/cmake/visp/VISPConfig-version.cmake  # [unix]
+    - test -f $PREFIX/lib/lib{{ name }}_core${SHLIB_EXT}  # [unix]
+    - if not exist %LIBRARY_PREFIX%\include\visp3\visp.h exit 1  # [win]
+    - if not exist %LIBRARY_PREFIX%\cmake\VISPConfig.cmake exit 1  # [win]
+    - if not exist %LIBRARY_PREFIX%\cmake\VISPConfig-version.cmake exit 1  # [win]
```

Nit: and I'll stop there. 
```suggestion
    - test -f $PREFIX/include/visp3/visp.h                                 # [unix]
    - test -f $PREFIX/lib/cmake/visp/VISPConfig.cmake                      # [unix]
    - test -f $PREFIX/lib/cmake/visp/VISPConfig-version.cmake              # [unix]
    - test -f $PREFIX/lib/lib{{ name }}_core${SHLIB_EXT}                   # [unix]
    - if not exist %LIBRARY_PREFIX%\include\visp3\visp.h exit 1            # [win]
    - if not exist %LIBRARY_PREFIX%\cmake\VISPConfig.cmake exit 1          # [win]
    - if not exist %LIBRARY_PREFIX%\cmake\VISPConfig-version.cmake exit 1  # [win]
```